doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,540 @@
+# Getting started with iceoryx
```

Comment:
I'd propose to split this file into three files:

* Overview / Who is who
* Core concepts (#API)
* Using the API / Basic API usage

This can be done before v1.0, when using this content for the website.

Reply:
Sure, we need to think how to arrange and distribute the information. For the 0.9 it is sufficient as it is I think.

For 1.0 I will improve the documentation.

Reply:
Ok, in the meantime. I'll try out an documentation export and push it to https://github.com/eclipse-iceoryx/iceoryx-web